python/llm/test/benchmark/csv_to_html.py

```diff
@@ -0,0 +1,15 @@
+# Python program to convert 
+# CSV to HTML Table
+ 
+import os
+import pandas as pd
+
+folder_path = "../../dev/benchmark/all-in-one"
```

Comment:
Better not hardcode the path, let's input the csv file as an argument?

Reply:
ok